example/lib/main.dart

```diff
@@ -11,6 +12,39 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MainPage();
+    return MaterialApp(
+      theme: ThemeData.light(),
+      darkTheme: ThemeData.dark(),
```

Comment:
Seems duplicate.

Reply:
I can remove `theme`, but if we remove `darkTheme`, the app won't have dark mode.

Reply:
And thanks for the quick review 🙂